incydr/cli/cmds/cases.py

```diff
@@ -46,17 +71,37 @@ def render_case(case: CaseDetail):
     console.print(Panel.fit(field_table))
 
 
-@app.command()
+def render_file_event(event: FileEventV2):
+    field_table = Table.grid(padding=(0, 1), expand=False)
+    field_table.title = f"File Event {event.event.id}"
+    event_dict = flatten(event.dict())
+    for name, _field in event_dict.items():
+        if name == "event.id":
+            continue
+        field_table.add_row(f"{name} = {event_dict[name]}")
+    console.print(Panel.fit(field_table))
+
+
+@cases.command(cls=IncydrCommand)
+@click.argument("name")
+@click.option("--description", help="Case description.", default=None)
+@click.option("--subject", help="UserID of the subject of the case.", default=None)
```

Comment:
It'd be nice if this could be a username too, so end users don't have to do a lookup first.